testssl.sh

```diff
@@ -760,8 +760,17 @@ fileout() {
      if ( "$do_pretty_json" && [[ "$1" == "service" ]] ) || show_finding "$severity"; then
          local finding=$(strip_lf "$(newline_to_spaces "$(strip_quote "$3")")")
          [[ -f "$JSONFILE" ]] && (fileout_json_finding "$1" "$severity" "$finding" "$cve" "$cwe" "$hint")
-         "$do_csv" && \
-              echo -e \""$1\"",\"$NODE/$NODEIP\",\"$PORT"\",\""$severity"\",\""$finding"\",\""$cve"\",\""$cwe"\",\""$hint"\"" >> "$CSVFILE"
+
+         if "$do_csv"; then
+            if "$do_test_just_one"; then
```

Comment:
a boolean variable "do_test_just_one" will be set to true only in the case of -x:

```
-x|-x=*|--single[-_]cipher|--single[-_]cipher=*)
     do_test_just_one=true
     single_cipher=$(parse_opt_equal_sign "$1" "$2")
     [[ $? -eq 0 ]] && shift
     ;;
```
 Is it a right name for it? How i understand if we type "-x CBC" -> all ciphers with cipher block chaining will be analysed. Maybe could we rename it to something like "only_specific_ciphers"? Otherwise "if "$do_test_just_one"; then" is abit confusing in my opinion by reading of the code.